packages/ui/src/lib/theme/defaultTheme.ts

```diff
@@ -470,6 +470,8 @@ export default {
       text-scale-1200
       border
       focus:shadow-md
+      dark:bg-[#292929]
+      focus:ring-[#292929]
```

Comment:
we may need to use actual tailwind colors we have or just reference css variables directly rather than hardcoding hex colors in.

will come back soon with some suggestions

Reply:
ok cool.
we can also use actual tailwind color like "trueGray-800".